mk/submodule.mk

```diff
@@ -54,11 +54,23 @@ endif
 	cd $(LIBRABBITMQ_PATH)/build && \
          cmake --build .
 
+# sse2neon related variables
+SSE2NEON_PATH = deps/sse2neon
+SSE2NEON_INCLUDE := -I $(SSE2NEON_PATH)
+
+$(SSE2NEON_PATH)/sse2neon.h:
+	git submodule update --init $(SSE2NEON_PATH)
+
 # Submodules
 SUBS := $(LIBTUV_PATH)/include
 ifeq ($(BUILD_REMOTE),1)
     SUBS += $(LIBRABBITMQ_PATH)/librabbitmq
 endif
+ifeq ($(UNAME_M),arm)
```

Comment:
Can you merge two `ifeq` conditions? Use `-o`.

Reply:
I'll merge them with `ifeq ($(UNAME_M),$(filter $(UNAME_M),arm aarch64))`.
Makefile does not support `-o` like shell script.

Reply:
Done.